### PR TITLE
[Qt] Simplify firmware libraries searching

### PR DESCRIPTION
**Pull request by scribam**

### CODE REVIEW

**Comment by Megamouse** [comment]
looks okay to me.

**Comment by Nekotekina** [comment]
I probably won't merge it because it uses QDir which is asking for trouble in future.

**Comment by scribam** [comment]
@Nekotekina I removed the part which used QDir.

File: rpcs3/rpcs3qt/settings_dialog.cpp [hunk 1]
```diff
@@ -280,50 +280,28 @@ settings_dialog::settings_dialog(std::shared_ptr<gui_settings> xSettings, const
...
-			}
+			list.append(ui->lleList->item(i)->clone());
 		}
 
-		// sort sublists
-		auto qLessThan = [](QListWidgetItem *i1, QListWidgetItem *i2) { return i1->text() < i2->text(); };
-		qSort(checked_Libs.begin(), checked_Libs.end(), qLessThan);
-		qSort(unchecked_Libs.begin(), unchecked_Libs.end(), qLessThan);
+		// sort list
+		qSort(list.begin(), list.end(), [](QListWidgetItem *i1, QListWidgetItem *i2) {
```
Comment (danilaml) [inline rpcs3/rpcs3qt/settings_dialog.cpp]: Please DON'T use `qSort()`. `QAlgorithms` have been depreacted for quite some time. Just use the std ones (i.e. in this case just a regular `std::sort`).

**Pull request merged**